### PR TITLE
Support chunked createOptions

### PR DESCRIPTION
**Pull request by LazarusX**

### CODE REVIEW

File: iotedgehubdev/composeproject.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 COMPOSE_VERSION = 3.6
 
+CREATE_OPTIONS_MAX_CHUNKS = 7
```
Comment (adashen) [inline iotedgehubdev/composeproject.py]: maybe 100 please ref https://github.com/Azure/iotedge/pull/396/files#diff-a23fc3bfea535e05dfedf483df4a8adeL96

File: iotedgehubdev/composeproject.py [hunk 2]
```diff
@@ -182,3 +184,21 @@ def my_unicode_repr(self, data):
...
+        if 'createOptions' not in settings:
+            return ''
+
+        res = settings['createOptions']
+
+        i = 0
+        while True:
+            i += 1
+            key = 'createOptions{0:0=2d}'.format(i)
+            if i <= CREATE_OPTIONS_MAX_CHUNKS and key in settings:
```
Comment (adashen) [inline iotedgehubdev/composeproject.py]: <= or <

File: iotedgehubdev/composeproject.py [hunk 3]
```diff
@@ -33,7 +35,7 @@ def compose(self):
         modules.update(self.module_content['$edgeAgent']['properties.desired']['modules'])
         for service_name, config in modules.items():
             self.Services[service_name] = {}
-            create_option_str = config['settings']['createOptions']
+            create_option_str = ComposeProject._join_create_options(config['settings'])
```
Comment (adashen) [inline iotedgehubdev/composeproject.py]: also for system modules

Comment (LazarusX) [reply iotedgehubdev/composeproject.py]: System modules (i.e., edgeHub) is already covered (https://github.com/Azure/iotedgehubdev/pull/99/files#diff-11647d7f78ebc8ad0d6859d0ff403c5bR35)

**Pull request merged**